client/main.js

```diff
@@ -12,39 +12,38 @@ function Event({ event }) {
   const eventDescription = desc => ({ __html: desc.replace(/\n/g, '<br>')});
   const eventCategory = event.info && event.info.Kategorija ? event.info.Kategorija : null;
 
-  return <div className={ `event list-group-item ${ eventCategory ? eventCategory.toLowerCase() : null }` }>
-    <h2>{ event.title } <EventTime time={event.time} /> </h2>
+  return <li className={ `event list-group-item ${ eventCategory ? eventCategory.toLowerCase() : null }` }>
+    <h2>{ event.title }</h2>
     <p>
-      <h4>{ event.info && event.info.Lokacija ? event.info.Lokacija : null }</h4>
+      <h4>
+        { event.info && event.info.Lokacija ? event.info.Lokacija : null }
+        &nbsp;
+        <EventTime time={event.time}/>
+      </h4>
       <span className="label label-primary">{ eventCategory }</span>
     </p>
     <p dangerouslySetInnerHTML={ eventDescription(event.description) }></p>
     <EventLink link={ event.link } />
-  </div>;
+  </li>;
 }
 
 function Date({ date, events }) {
-  const eventList = events.map(event => <Event event={event} />);
   const dateChunks = date.split('/');
   const normalizedDateString = `${dateChunks[1]}/${dateChunks[0]}`
   const formatedDate = format(normalizedDateString, 'D. MMMM', { locale: hrLocale })
 
-  return <div className="date list-group">
-    <h1>
-      { formatedDate }
-    </h1>
-    <div>
-      { eventList }
-    </div>
-  </div>
+  return <div className="date">
+    <h1>{ formatedDate }</h1>
+    <ul className="list-group">
+      { events.map(event => <Event event={ event } />) }
+    </ul>
+  </div>;
 }
 
 function Dates(dates) {
-  const dateList = dates.map(({ date, events }) => <Date date={date} events={events} />)
-
   return <div className="dateList container">
-    <div>{ dateList }</div>;
-  </div>
+    { dates.map(({ date, events }) => <Date date={ date } events={ events } />) }
+  </div>;
```

Comment:
Um this is general comment but it is something that is both easy enough to do in one shot and will increase code readability. Let's reformat JSX chunk to follow react convention, a-la: [code](https://prettier.io/playground/#%7B%22content%22%3A%22function%20Dates(dates)%20%7B%5Cn%20%20return%20%3Cdiv%20className%3D%5C%22dateList%20container%5C%22%3E%5Cn%20%20%20%20%7B%20dates.map((%7B%20date%2C%20events%20%7D)%20%3D%3E%20%3CDate%20date%3D%7B%20date%20%7D%20events%3D%7B%20events%20%7D%20%2F%3E)%20%7D%5Cn%20%20%3C%2Fdiv%3E%3B%5Cn%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D)